src/loaders/logger.ts

```diff
@@ -0,0 +1,34 @@
+import winston from 'winston';
+import config from '../config';
+
+const transports = [];
+if(process.env.NODE_ENV !== 'development') {
```

Comment:
linter :)
developpement -> dev

Reply:
et utilise plutot la forme positive du if

```suggestion
if (process.env.NODE_ENV === 'dev') {
```